markdown.go

```diff
@@ -189,6 +190,7 @@ type Renderer interface {
 	TripleEmphasis(out *bytes.Buffer, text []byte)
 	StrikeThrough(out *bytes.Buffer, text []byte)
 	FootnoteRef(out *bytes.Buffer, ref []byte, id int)
+	Math(out *bytes.Buffer, equation []byte, inline bool)
```

Comment:
I don't think we can accept a change that changes the `Renderer` interface at this stage, as it would invalidate all existing `blackfriday.Renderer` implementations.